### DISCUSSION TITLE
wecom 企业微信登录taro如何授权登录

### DISCUSSION THREAD

**Discussion by twanglvy** [NervJS/taro#15196] [Q & A]
### 这个特性解决了什么问题？
wecom 企业微信登录taro如何授权登录

### 这个 API 长什么样？
taro.wx.qy.login

**Comment by ZakaryCode** [comment]
直接 wx.qy.login 即可

**Reply by twanglvy** [reply]
微信环境下怎样使用企业微信授权登录
